### PR TITLE
Corrected script for some item

### PR DESCRIPTION
**Pull request by idk-whoami**
* **Addressed Issue(s)**: 

* **Server Mode**: re

* **Description of Pull Request**: 
Data correction for item combo IDs 450144:1846

https://www.divine-pride.net/database/item/1846
https://www.divine-pride.net/database/item/450144

Data correction for item IDs 4367, 4634, 4979, 2006, 5414, 19146, 24326, 24327, 24334, 24335, 24336, 24360, 24361, 24362, 24363, 24364, 24365, 24378, 24380, 24381, 24382, 24383, 24384, 24385, 24396, 24402, 24403, 24404, 24405, 24406, 24407, 24408, 24409, 24410, 24411, 24426, 24432, 24433, 24443, 24454

https://www.divine-pride.net/database/item/4367
https://www.divine-pride.net/database/item/4634
https://www.divine-pride.net/database/item/4979
https://www.divine-pride.net/database/item/2006
https://www.divine-pride.net/database/item/5414
https://www.divine-pride.net/database/item/19146
https://www.divine-pride.net/database/item/24326
https://www.divine-pride.net/database/item/24327
https://www.divine-pride.net/database/item/24334
https://www.divine-pride.net/database/item/24335
https://www.divine-pride.net/database/item/24336
https://www.divine-pride.net/database/item/24360
https://www.divine-pride.net/database/item/24361
https://www.divine-pride.net/database/item/24362
https://www.divine-pride.net/database/item/24363
https://www.divine-pride.net/database/item/24364
https://www.divine-pride.net/database/item/24365
https://www.divine-pride.net/database/item/24378
https://www.divine-pride.net/database/item/24380
https://www.divine-pride.net/database/item/24381
https://www.divine-pride.net/database/item/24382
https://www.divine-pride.net/database/item/24383
https://www.divine-pride.net/database/item/24384
https://www.divine-pride.net/database/item/24385
https://www.divine-pride.net/database/item/24396
https://www.divine-pride.net/database/item/24402
https://www.divine-pride.net/database/item/24403
https://www.divine-pride.net/database/item/24404
https://www.divine-pride.net/database/item/24405
https://www.divine-pride.net/database/item/24406
https://www.divine-pride.net/database/item/24407
https://www.divine-pride.net/database/item/24408
https://www.divine-pride.net/database/item/24409
https://www.divine-pride.net/database/item/24410
https://www.divine-pride.net/database/item/24411
https://www.divine-pride.net/database/item/24426
https://www.divine-pride.net/database/item/24432
https://www.divine-pride.net/database/item/24433
https://www.divine-pride.net/database/item/24443
https://www.divine-pride.net/database/item/24454

### CODE REVIEW

File: db/re/item_db_equip.yml [hunk 1]
```diff
@@ -102334,7 +102314,6 @@ Body:
     Refineable: true
     Script: |
       .@r = getrefine();
-      bonus bMaxHP,.@r*10;
```
Comment (Atemo) [inline db/re/item_db_equip.yml:102337]: This bonus is written in the aegis data but not in the description.

Actually I grep the shadow items in the aegis data : **every single one of them** have the bonus
```
bonus bBaseAtk,getrefine();
bonus bMatk,getrefine();
```
for shadow weapon and 
```
bonus bMaxHP,getrefine()*10;
```
for the others equipments, even if they have sometimes omitted the bonus in the description. I guess all the shadow eq should have the bonus of hp/atk by refine.

To resume: for me all our shadow equipment should have the extra bonus of atk/hp regardless of the description.

Comment (idk-whoami) [reply db/re/item_db_equip.yml]: Thanks

File: db/re/item_db_etc.yml [hunk 2]
```diff
@@ -11609,11 +11609,9 @@ Body:
       BuyingStore: true
       DropEffect: CLIENT
     Script: |
+      bonus bCritical,9;
       if (BaseLevel<100) {
-         bonus bCritical,9;
-      }
-      else {
-         bonus bCritical,10;
+         bonus bCritical,1;
```
Comment (Atemo) [inline db/re/item_db_etc.yml:11614]: For me the previous item script was fine
```
CRI + 9.
At base level 100 or higher, CRI + 1 added, increases critical damage by 5%. 
```

File: db/re/item_db_equip.yml [hunk 3]
```diff
@@ -19768,8 +19765,9 @@ Body:
     EquipLevelMin: 60
     Refineable: true
     Script: |
-      bonus bAtkEle,Ele_Holy;
       bonus bInt,6;
+          bonus bMaxSP,150;
+          bonus bMatkRate,18;
```
Comment (Atemo) [inline db/re/item_db_equip.yml:19770]:
```suggestion
      bonus bMaxSP,150;
      bonus bMatkRate,18;
```

File: db/re/item_db_equip.yml [hunk 4]
```diff
@@ -43827,7 +43825,7 @@ Body:
       NoAuction: true
     Script: |
       bonus2 bSubEle,Ele_Fire,10;
-      bonus4 bAutoSpellWhenHit,"MG_FIREBALL",5,500,1;
+      bonus4 bAutoSpellWhenHit,"MG_FIREBALL",5,30,1;
```
Comment (Atemo) [inline db/re/item_db_equip.yml:43828]: Maybe our data is outdated but it's 500 in our aegis file, please revert
```suggestion
      bonus4 bAutoSpellWhenHit,"MG_FIREBALL",5,500,1;
```

File: db/re/item_db_equip.yml [hunk 5]
```diff
@@ -101149,7 +101147,7 @@ Body:
       .@r = getrefine();
       bonus bLuk,1;
       bonus bHit,.@r;
-      if (BaseClass == Job_Swordman || BaseClass == Job_Thief || (BaseClass == Job_Taekwon && BaseJob != Job_Soul_Linker)) {
+      if (BaseClass == Job_Swordman || BaseClass == Job_Thief || (BaseClass == Job_Taekwon && BaseJob != Job_Star_Gladiator)) {
```
Comment (Atemo) [inline db/re/item_db_equip.yml:101150]: You misread, it was fine before
```
If equipped by Swordsman classes, Thief classes, Taewon Kid and Star Gladiator classes, Atk + 15.
```
```suggestion
      if (BaseClass == Job_Swordman || BaseClass == Job_Thief || (BaseClass == Job_Taekwon && BaseJob != Job_Soul_Linker)) {
```

File: db/re/item_db_equip.yml [hunk 6]
```diff
@@ -101171,7 +101169,7 @@ Body:
       .@r = getrefine();
       bonus bVit,1;
       bonus bDef,5*.@r;
-      if (BaseClass == Job_Swordman || BaseClass == Job_Thief || (BaseClass == Job_Taekwon && BaseJob != Job_Soul_Linker)) {
+      if (BaseClass == Job_Swordman || BaseClass == Job_Thief || (BaseClass == Job_Taekwon && BaseJob != Job_Star_Gladiator)) {
```
Comment (Atemo) [inline db/re/item_db_equip.yml:101172]:
```suggestion
      if (BaseClass == Job_Swordman || BaseClass == Job_Thief || (BaseClass == Job_Taekwon && BaseJob != Job_Soul_Linker)) {
```

File: db/re/item_db_equip.yml [hunk 7]
```diff
@@ -103014,7 +103012,7 @@ Body:
     EquipLevelMin: 99
     Refineable: true
     Script: |
-      bonus2 bSkillAtk,"RA_CLUSTERBOMB",5+3*(getrefine()/2);
+      bonus2 bSkillAtk,"RA_CLUSTERBOMB",5+2*(getrefine()/2);
```
Comment (Atemo) [inline db/re/item_db_equip.yml:103015]: It was fine, 3% per 2 refine (KRO description)
```suggestion
      bonus2 bSkillAtk,"RA_CLUSTERBOMB",5+3*(getrefine()/2);
```

**Review by Atemo — CHANGES_REQUESTED** [review]

File: db/re/item_db_etc.yml (hunk 2 above)
Comment (idk-whoami) [reply db/re/item_db_etc.yml]: Thanks

File: db/re/item_db_equip.yml (hunk 3 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml]: Thanks

File: db/re/item_db_equip.yml (hunk 4 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml]: Thanks

File: db/re/item_db_equip.yml (hunk 5 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml]: Thanks

File: db/re/item_db_equip.yml (hunk 6 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml]: Thanks

File: db/re/item_db_equip.yml (hunk 7 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml]: Thanks

**Comment by Atemo** [comment]
Thanks !

**Pull request merged**